src/Libraries/CoreNodeModels/Properties/Resources.en-US.resx

```diff
@@ -397,13 +397,13 @@
     <value>operand</value>
   </data>
   <data name="PortDataResultToolTip" xml:space="preserve">
-    <value>result</value>
+    <value>Result block produced</value>
   </data>
   <data name="PortDataTestBlockToolTip" xml:space="preserve">
-    <value>Test block</value>
+    <value>Boolean to determine what code block is returned</value>
```

Comment:
code block is confusing as it sounds like its returning a `CodeBlock`, is this for the `scopeIf` node?